AzureMonitorAgent/agent.py

```diff
@@ -464,8 +464,12 @@ def enable():
             log_and_exit("Enable", MissingorInvalidParameterErrorCode, 'Mixing genevaConfiguration or azureMonitorConfiguration with other configuration schemas is not allowed')
 
         if geneva_configuration and geneva_configuration.get("enable") == True:
-            hutil_log_info("Detected Geneva+ mode; azuremonitoragentmgr service will be started to handle Geneva tenants")
             ensure["azuremonitoragentmgr"] = True
+            if azure_monitor_configuration and azure_monitor_configuration.get("enable") == True:
```

Comment:
this log change is unnecessary, we log wheter AMCS mode is also enabled below on original line 471. please revert to the original, and only add the `generate_localsyslog_configs()` to this block  (block starting line 466)